src/cobertura.rs

```diff
@@ -246,13 +249,12 @@ fn get_coverage(
             }
             start_indexes.sort_unstable();
 
-            let functions = result.functions;
-            let result_lines = result.lines;
-            let result_branches = result.branches;
+            // let result_lines = result.lines.clone();
+            // let result_branches = result.branches.clone();
```

Comment:
Can you remove these commented out lines?

Reply:
removed